libs/zgui/src/gui.zig

```diff
@@ -1,4 +1,19 @@
 //--------------------------------------------------------------------------------------------------
+//
+// Zig bindings for 'dear imgui' library. Easy to use, hand-crafted API with default arguments,
+// named parameters and Zig style text formatting.
+//
+//--------------------------------------------------------------------------------------------------
+pub const version = @import("std").SemanticVersion{ .major = 0, .minor = 9, .patch = 6 };
+
+pub usingnamespace @import("gui.zig");
```

Comment:
Does this work? File exports itself?

Reply:
Whoops! I meant to remove that line, just pushed the change.